examples/data-objects/inventory-app/src/schema.ts

```diff
@@ -3,24 +3,22 @@
  * Licensed under the MIT License.
  */
 
-import { FieldKinds, SchemaAware, SchemaBuilder, ValueSchema } from "@fluid-experimental/tree2";
+import { FieldKinds, SchemaBuilder, TypedField, TypedNode, leaf } from "@fluid-experimental/tree2";
 
-const builder = new SchemaBuilder("inventory app");
-export const float64 = builder.leaf("number", ValueSchema.Number);
-export const string = builder.leaf("string", ValueSchema.String);
+const builder = new SchemaBuilder("inventory app", {}, leaf.library);
 
 export const part = builder.struct("Contoso:Part-1.0.0", {
-	name: SchemaBuilder.field(FieldKinds.value, string),
-	quantity: SchemaBuilder.field(FieldKinds.value, float64),
+	name: SchemaBuilder.field(FieldKinds.value, leaf.string),
+	quantity: SchemaBuilder.field(FieldKinds.value, leaf.number),
```

Comment:
'.fieldValue'?

Reply:
We could do that (means the same thing). This is all getting rewritten when we do the new schema builder API, which should fix that we have two separate patterns for this, so it I'll just leave it for now.